docs/kratos/self-service/flows/account-recovery-password-reset.mdx

```diff
@@ -0,0 +1,237 @@
+
+## Updating credentials
+
+After the user completes the recovery successfully they receive a privileged session and are redirected to the settings page to
+update their credentials.
+
+Read more about the [privileged sessions](../../session-management/01_overview.mdx#privileged-sessions).
+
+## Invalidate other sessions
+
+In some scenarios it can be useful to revoke all active sessions once an account has been recovered. This forces anyone with
+access to the account to re-authenticate.
+
+```diff
+ selfservice:
+   flows:
+     recovery:
+       enabled: true
+       ui_url: http://127.0.0.1:4455/recovery
++      after:
++        hooks:
++        - hook: revoke_active_sessions
+```
+
+## One-time passwords vs. magic links
+
+The one-time password method (`code`) is preferred, as magic links have multiple drawbacks:
+
+- Some email virus scanners open links in emails to scan them. This invalidates the magic link and a user may not be able to
+  recover their account.
+- API flows initialized by apps on mobile phones or smart devices are not possible when using magic links.
+- Magic links often open in different browsers than the one used to initialize the flow, depending on the user's device settings.
+  This may result in a confusing user experience for your users.
+
+Ory Identities (Ory Kratos) uses one-time passwords by default. Magic links are available for legacy reasons and will be removed
+in the future.
+
+:::important
+
+When updating an existing project from the `link` method to use the `code` method, there might be changes in the behavior of the
+UI. Double-check the flow after updating your configuration.
+
+:::
+
+## Multifactor authentication
+
+By design, the account recovery process bypasses the primary account credential. You can choose to require multifactor
+authentication for account recovery.
+
+<Tabs
+  defaultValue="oc"
+  values={[
+    {label: 'The Ory Network', value: 'oc'},
+    {label: 'Self-hosted Ory Kratos config', value: 'oss'}
+  ]}>
+  <TabItem value="oc">
+On the Ory Network Console go to <b>Two-Factor Authentication</b> and disable "Allow Self-Service Settings without Second Factor".
+  </TabItem>
+  <TabItem value="oss">
+
+```yaml title="path/to/my/kratos/config.yml"
+selfservice:
+  flows:
+    settings:
+      required_aal: highest_available
+```
+
+  </TabItem>
+</Tabs>
+
+## Code examples for Node.js, React.js, Go
+
+The recovery user interface is a page in your own application (server, native app, single page app) that should render the actual
+form elements for the user.
+
+In contrast to other identity systems, Ory Identities (Ory Kratos) doesn't render this HTML directly. Instead, you need to
+implement the HTML code in your application (for example in Node.js + Express.js, Java, PHP, React.js, ...), which gives you
+complete flexibility and customizability in your user interface flows and designs. This part of your application then directly
+interfaces with Ory Identities through an API.
```

Comment:
```suggestion
In contrast to other identity systems, Ory Identities (Ory Kratos) doesn't render this HTML directly. Instead, you need to
implement the HTML code in your solution, which gives you
complete flexibility and customizability in your user interface flows and designs. This part of your application then directly
interfaces with Ory Identities through the API.
```

Reply:
Changed "an API" to "the API" - it's about Ory API, right?

Reply:
Yes, "the API" is also okay. 

We do have multiple APIs depending on the use-case, but in general we are refering to all Ory APIs here.